[Content] Korath Exiles Expansion Preparation: Raider Update (#6870)
endless-sky/endless-sky: data/korath/korath ships.txt

	explode "large explosion" 40
	explode "huge explosion" 10
	"final explode" "final explosion large"

> 	description "The Korath Exiles designed the Raider soon after the Dredger. The Raider can be built cheaply in bulk, allowing for non-stop raids of nearby species."

Reviewer: Not sure I buy this desc if this is actually a Mark 41.  And, I thought I'd throw in a lore bone to the classification...
```suggestion
	description "Ships like the Raider have been the Korath warship of choice for plundering neighbouring species for generations."
	description "This model, like all Exile ships, has been adapted to house as many Korath as possible, as comfortably as possible, inflating its size beyond similarly classed warships. The spaceborne construction techniques employed allow for easy replacement, facilitating continued raids of nearby systems."
```

Reviewer 2: That looks good; although it might be a little all-encompassing. "ships like the Raider" could be a pretty big set. Perhaps it would be better to focus that in a little starting it off with something like "Iterations of the Raider design have been the..." or perhaps "Variations of the Raider design have been the..." That makes it clear that there have been many versions of it over the years, and we're not just talking about a general category of ships.

Reviewer: The vagueness was intentional - it could be a big set, it might not be.  However, if specificity is desired here, how's this?  I'm trying to keep it from being wordy:
```suggestion
	description "Successive Raider designs have been the Korath warship of choice for plundering neighbouring species for generations."
	description "This model, like all Exile ships, has been adapted to house as many Korath as possible, as comfortably as possible, inflating its size beyond similarly classed warships. The spaceborne construction techniques employed allow for easy replacement, facilitating continued raids of nearby systems."
```

Reviewer 2: That looks good!

	explode "large explosion" 40
	explode "huge explosion" 10
	"final explode" "final explosion large"
	description "Successive Raider designs have been the Korath warship of choice for plundering neighboring species for generations."

> 	description "This model, like all Exile ships, has been adapted to house as many Korath as possible, as comfortably as possible, inflating its size beyond similarly classed warships. The spaceborne construction techniques employed allow for easy replacement, facilitating continued raids of nearby systems."

Reviewer 3: ```suggestion
	description "This model, like all Exile ships, has been adapted to house as many Korath as possible, as comfortably as possible, inflating its size beyond similarly classed warships. The spaceborne construction techniques employed allow for easy replacement of broken or outdated equipment, facilitating continued raids of nearby systems."
```

Reviewer 3: Ah ok, how about this then?
```suggestion
	description "This model, like all Exile ships, has been adapted to house as many Korath as possible, as comfortably as possible, inflating its size beyond similarly classed warships. The spaceborne construction techniques employed allow for the ship's easy replacement, facilitating continued raids of nearby systems."
```

Reviewer: Sure, that works

		left
	"steering engine" 89 33.5
		zoom 0.3
		angle -126
		right
	"steering engine" -82.5 30
		zoom 0.3
		angle 126
		left
	"steering engine" 82.5 30
		zoom 0.3
		angle -126
		right
	"steering engine" -69.5 -103.5
		zoom 0.3
		angle 38
		right
	"steering engine" 69.5 -103.5
		zoom 0.3
		angle -38
		left
	"steering engine" -76 -106
		zoom 0.3
		angle 38
		right
	"steering engine" 76 -106
		zoom 0.3
		angle -38
		left
	gun -45 -150 "Korath Fire-Lance"
	gun 32.5 -1
		angle 60
	gun -32.5 -1
		angle 300
	turret -10 -143 "Korath Grab-Strike"
	turret 10 -143 "Korath Grab-Strike"
	turret -36 -141 "Korath Grab-Strike"
	turret 36 -141 "Korath Grab-Strike"
	turret 0 41 "Korath Banisher"
	turret 0 94 "Korath Warder"
	bay "Fighter" -80 151 back
	bay "Fighter" 80 151 back
	bay "Fighter" 28.5 145 back
	explode "tiny explosion" 20
	explode "small explosion" 45
	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 10
	"final explode" "final explosion large"
	description "The venerable Raider has long been a mainstay of the Korath raiding fleet, originally built and manufactured during an era of shortages and desperation as they adapted to life in exile."

> 	description `In the ensuing centuries, however, constant warfare against surrounding aliens has been building pressure on their shipyards to recover some of their ancient prowess. Now, with a fresh industrial network to support their endeavors, the shipwrights of the Korath have revised the design to bring it up to their expectations.`

Reviewer 4: The timeline is that the Korath Exiles have been exiled for a century. This doesn't add up to that.

Author: Is decades more appropriate?

Reviewer 4: Decades or years would fit best.

I'm also not sure it's good to lore the Raider to have been developed in the early Exile and not before then, as that'd rule it out as a ship used to raid the Hai. We know the Hai had to fend off Korath raids, and from what's in the game about that it's not implied that they were helplessly getting washed over by the Korath, so we can assume the Korath weren't using their T2 level ships and tech, as those, either Mereti or Sestor, would wash over the Hai.
It was my understanding that the Raider was the ship employed then, which makes the Hai managing to defend themselves more believable, since the Raider is much more manageable for Shield Beetles to fight.

Author: description `In the ensuing decades, however, constant warfare against surrounding aliens has been building pressure on their shipyards to recover some of their ancient prowess. Now, with a fresh industrial network to support their endeavors, the shipwrights of the Korath have revised the design to bring it up to their expectations.`
	
Does this sound more appropriate?

Reviewer: > The timeline is that the Korath Exiles have been exiled for a century. This doesn't add up to that.

A century singular?  Where is this noted?

In the timeline we've been using to develop Korath content, I put the Exodus around 2800, with the Raider being developed some 20 years later in 2820

Reviewer 5: I have the following dates on my timeline:
* Some time between 2400 to 2600: The Korath Civil War begins.
  * Wanderers: Kor Efret 4: "There was a civil war. It lasted hundreds of years." No way of telling exactly how long "hundreds" is.
* Some time in the 2700s: The Korath begin sending Raiders into Hai space, beginning the Korath-Hai War.
  * Ask the Hai about the Unfettered: "Hundreds of years ago, when I was young, our territory was raided by aliens from the east, who came in massive warships and plundered our worlds."
* Early 2800s: The Korath stop sending Raiders into Hai space.
  * Ask the Hai about the Unfettered: "The raids lasted for decades, then abruptly stopped."
  * Wanderers: Alpha Surveillance A: "Indeed, our first contact with the humans, two centuries ago, was with these green people... But then the peaceful humans came, and we learned that you are not all like the Alphas... And during the time of the Korath raids, we would not have had the capacity to track them, anyway." I gather from this that the Alphas showed up during the raids, but humans showed up after the raids.
* Early 2900s: The Korath are exiled by the Drak.
  * Wanderers: Kor Efret 4: "Eventually, about a century ago, it became clear that the war would consume and destroy both empires, if it continued. The Archons stepped in and gave a choice to each member of the Kor Mereti and the Kor Sestor."
  * FW Syndicate Extremists 1A: "Years ago, some leaders of our financial division were approached by some... consultants... The Korath raids had just begun, so we suddenly had access to alien technology. They were interested in all of it..."
  * Remnant: Defense 2: "They explain to you (in song, of course; the explanation takes the form of an epic ballad) that these raids have been taking place for nearly a century."

POSSIBLE CHANGES: I think it would be possible to move the first three events up 100 years. So 2400-2600 becomes 2500-2700, 2700s becomes 2800s, and then the raids stop at the same time that the Korath are exiled. I believe when I first wrote these down I assumed that the raids would have stopped as the war became more dire and the Korath didn't even have the resources remaining to send raids, but looking at the quotes again, the "The raids lasted for decades, then abruptly stopped," now sounds to me like they abruptly stopped because the Korath were no longer capable of sending raids because by that point they'd been exiled.

Reviewer: The dating of the Hai raids I don't find particularly important to this discussion, and I think they are fine where they are.

I think the Exodus can be moved up to the late 2800s though - it works with the "about a hundred years" statement.  I think they would have needed time to build up their habitats and new industrial base, exploit their local resources, and investigate their new surroundings for quite some time before deciding to raid the humans. Furthermore, the relatively primitive humans may not have been as attractive targets until more recently.

Psychological and political factors may have also temporarily stayed their claws, having just endured the Exodus they may have been wary of further angering the Drak.

So... I can see where the Raider may have been a very recent design, perhaps late 2900s.

Reviewer 5: > The dating of the Hai raids I don't find particularly important to this discussion.

I'd say it is important if the Raider was used during the Hai raids, as Arachi suggests and I think makes sense.

> Psychological and political factors may have also temporarily stayed their claws, having just endured the Exodus they may have been wary of further angering the Drak.

So a timeline more like as follows?
<Late 2700s: Korath Civil War begins.
Late 2700s: Raids on Hai space begin.
Late 2800s: Korath are exiled. Raids on Hai end.
Early 2900s: Korath begin raiding human and Remnant space.

Reviewer: > > The dating of the Hai raids I don't find particularly important to this discussion.
> 
> I'd say it is important if the Raider was used during the Hai raids, as Arachi suggests and I think makes sense.

The only justification he's given for that has been that the hai were being raided, so the ship currently named the raider must have been doing he raiding?  That seems silly to me.  Any ship can raid without being called a raider, and the name raider doesn't have to apply to one singular design for several hundred years -- see below about the idea of Raider being a class that may be fulfilled by different designs in different eras.

Maybe the Hai would do that, but I don't see the Korath being that stagnant, especially not after enduring such a radical change to their society as the Exodus.

Name aside, I don't think it makes sense for the Korath to still be using a centuries old, pre-exile design as a front line combat unit.

> > Psychological and political factors may have also temporarily stayed their claws, having just endured the Exodus they may have been wary of further angering the Drak.
> 
> So a timeline more like as follows? <Late 2700s: Korath Civil War begins. Late 2700s: Raids on Hai space begin. Late 2800s: Korath are exiled. Raids on Hai end. Early 2900s: Korath begin raiding human and Remnant space.

Haw about...

2600s - Korath Civil Wars begin, society fractures into 4-5 factions
2700s - Some Korath factions begin sending raids into their respective adjoining spaces, notably to the Hai
Mid-Late 2800s - Korath factions detonate nova bombs against each other, Drak step in, Hai raids stop, Exodus happens
Early 2900s - Exiles are busy building up a new industrial base, Efreti are getting friendly with the Quarg
Undetermined time in the 2900s - The last of the living Mereti die out, Exiles begin raiding Remnant space
Late 2900s - As yet unnamed faction in the northwest of Korath space dies out.  Exiles begin raiding Human space

Author: Feel free to continue this discussion, though, for the sake of having the PR being analyzed for its focuses and not on a description that A- nobody will see until way later when Raiders are available somehow in a Korath Campaign, and B- will very likely change description again at that point, I will just remove the word "long" from "long before exile life", and leave it ambiguous. 

~~I don't want another Bactrian PR stuck in purgatory situation due to a part that isn't the focus again~~

	turret "Korath Warder"

ship "Korath Raider Mark XLII"
	sprite "ship/raiderii"
	thumbnail "thumbnail/raiderii"
	attributes
		category "Medium Warship"
		"cost" 22395000
		"shields" 27500
		"hull" 11000
		"required crew" 157

> 		"bunks" 250

Reviewer 6: with the addition of the gun port shuttle thing below, shouldn't we have a bit more bunks? Maybe just 5 extra, but I feel like there should be some more.
Also maybe add 500 more shields?

Reviewer: I think the bunk count is fine. And shields are already buffed over the extant Raider.

Author: The idea behind not buffing bunks is although the Raider Mark XLII is a newer model of the Raider, it's not an increased sized raider. It has more functionality, more modules, but not more room for crew. Think of adding extra arsenal to WWII plane- additional gun pods and cannons under the wings, but still the same frame and pilot count.

Shields were already buffed by 500, armor by 1000. It makes sense for these ratios based on the visual update to the design.

	turret "Korath Warder"

ship "Korath Raider Mark XLII"
	sprite "ship/raiderii"
	thumbnail "thumbnail/raiderii"
	attributes
		category "Medium Warship"
		"cost" 22395000

> 		"shields" 27500

Reviewer 2: ```suggestion
		"shields" 29500
```

Let's give them an actual buff - not just a polish. With my suggested change here, that brings them up to a +9.2% shield capacity boost instead of the previous +1.8%

		over
	engine 0 142
		over
	"reverse engine" -9 -154
		zoom 0.7
	"reverse engine" 9 -154
		zoom 0.7
	"steering engine" -89 33.5
		zoom 0.3
		angle 126
		left
	"steering engine" 89 33.5
		zoom 0.3
		angle -126
		right
	"steering engine" -82.5 30
		zoom 0.3
		angle 126
		left
	"steering engine" 82.5 30
		zoom 0.3
		angle -126
		right
	"steering engine" -69.5 -103.5
		zoom 0.3
		angle 38
		right
	"steering engine" 69.5 -103.5
		zoom 0.3
		angle -38
		left
	"steering engine" -76 -106
		zoom 0.3
		angle 38
		right
	"steering engine" 76 -106
		zoom 0.3
		angle -38
		left
	gun -45 -150 "Korath Fire-Lance"
	gun 32.5 -1 "Firelight Missile Bank"
		angle 60
	gun -32.5 -1 "Firelight Missile Bank"
		angle 300
	turret -10 -143 "Korath Grab-Strike"
	turret 10 -143 "Korath Grab-Strike"
	turret -36 -141 "Korath Grab-Strike"
	turret 36 -141 "Korath Grab-Strike"
	turret 0 41 "Korath Banisher"
	turret 0 94 "Korath Warder"
	bay "Fighter" -80 151 back
	bay "Fighter" 80 151 back
	bay "Fighter" 28.5 145 back
	explode "tiny explosion" 20
	explode "small explosion" 45
	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 10
	"final explode" "final explosion large"

> 	description "The venerable Raider has long been a mainstay of the Korath raiding fleet, designed before the era of shortages and desperation of life in exile."

Reviewer 3: ```suggestion
	description "The venerable Raider has long been a mainstay of the Korath raiding fleet, designed before the era of shortages and desperation caused by a life in exile."
```

		angle -38
		left
	gun -45 -150 "Korath Fire-Lance"
	gun 32.5 -1 "Firelight Missile Bank"
		angle 60
	gun -32.5 -1 "Firelight Missile Bank"
		angle 300
	turret -10 -143 "Korath Grab-Strike"
	turret 10 -143 "Korath Grab-Strike"
	turret -36 -141 "Korath Grab-Strike"
	turret 36 -141 "Korath Grab-Strike"
	turret 0 41 "Korath Banisher"
	turret 0 94 "Korath Warder"
	bay "Fighter" -80 151 back
	bay "Fighter" 80 151 back
	bay "Fighter" 28.5 145 back
	explode "tiny explosion" 20
	explode "small explosion" 45
	explode "medium explosion" 50
	explode "large explosion" 40
	explode "huge explosion" 10
	"final explode" "final explosion large"
	description "The venerable Raider has long been a mainstay of the Korath raiding fleet, designed before the era of shortages and desperation caused by a life in exile."
	description `Constant warfare against surrounding aliens has been building pressure on Korath shipyards to recover some of their ancient prowess. Now, with a fresh industrial network to support their endeavors, the shipwrights of the Korath have updated the design to deal with modern targets.`

ship "Korath Raider Mark XLII" "Korath Raider Mark XLII (Crippler)"
	sprite "ship/raiderxlii"

> 	thumbnail "thumbnail/raiderxlii"

Reviewer 7: You don't need to specify the sprite/thumbnail if they're the same as the base model.

Author: https://github.com/endless-sky/endless-sky/pull/6955

	add attributes
		"cargo space" -70

> 		"outfit space" 70

Reviewer 7: While we have special variants that modify base attributes (like the Barb, or Auxiliary), I don't think it makes sense for this variant to modify the base stats since it's literally the same ship (sprite) as the base model.

Although I'm not a content person, so...

Author: https://github.com/endless-sky/endless-sky/pull/6955 If you want to continue the conversation here.

	explode "large explosion" 40
	explode "huge explosion" 10
	"final explode" "final explosion large"

> 	description "The venerable Raider has long been a mainstay of the Korath raiding fleet, built and manufactured before the era of shortages and desperation in life of exile."

Reviewer: ```suggestion
	description "The venerable Raider has long been a mainstay of the Korath raiding fleet, designed before the current era of shortages and desperation of life in exile."
```
- raiders may still be produced
- this change leaves open the possibility of it being designed either before or during exile
- grammar

Author: Done! Ambiguity for the win!
